README.md

```diff
@@ -110,15 +117,16 @@ npx @airtasker/spot generate --contract api.ts
 # Commands
 
 <!-- commands -->
-* [`spot checksum SPOT_CONTRACT`](#spot-checksum-spot_contract)
-* [`spot docs SPOT_CONTRACT`](#spot-docs-spot_contract)
-* [`spot generate`](#spot-generate)
-* [`spot help [COMMAND]`](#spot-help-command)
-* [`spot init`](#spot-init)
-* [`spot lint SPOT_CONTRACT`](#spot-lint-spot_contract)
-* [`spot mock SPOT_CONTRACT`](#spot-mock-spot_contract)
-* [`spot validate SPOT_CONTRACT`](#spot-validate-spot_contract)
-* [`spot validation-server SPOT_CONTRACT`](#spot-validation-server-spot_contract)
+
```

Comment:
just confirming, is this just an extra space?

Reply:
yup, this stuff gets butchered during the release